.travis.yml

```diff
@@ -18,8 +17,6 @@ services:
 
 matrix:
   include:
-    - php: 5.4
-      env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
       env: SYMFONY_VERSION="~2.3.27 symfony/console:2.5.*"
```

Comment:
I would like to keep a ``prefer-lowest`` test, can you change this line to ``COMPOSER_FLAGS="--prefer-lowest"`` ?